lizard_auth_server/models.py

```diff
@@ -450,3 +450,19 @@ def __unicode__(self):
         else:
             return "{role} in {org}".format(
                 role=self.role, org=self.organisation)
+
+    def as_dict(self):
+        """
+        """
+        org = Organisation.objects.get(id=self.organisation.id)
+        role = Role.objects.get(id=self.role.id)
+        portal = Portal.objects.get(id=role.portal.id)
```

Comment:
Waarom wordt dit allemaal opgehaald, eigenlijk? `self.organisation` is toch al de organisatie? Idem `self.role.portal`, toch?